src/ApiPort.VisualStudio/AnalyzeMenu.cs

```diff
@@ -171,5 +184,23 @@ private ICollection<Project> GetSelectedProjects()
                 .Distinct()
                 .ToList();
         }
+
+        private ICollection<Project> GetTransitiveReferences(IEnumerable<Project> projects, HashSet<Project> expandedProjects = null)
```

Comment:
If the default case is `expandedProjects = null`, when does that get initialized? Is there a NullReferenceException when this method is called?